components/CandidateList/CandidateList.tsx

```diff
@@ -0,0 +1,20 @@
+﻿import React from "react";
+import scss from './CandidateList.module.scss';
+import {getCandidates} from "../../api/candidatesApiClient";
+import {CandidateLink} from "../CandidateLink/CandidateLink";
+
+
+export function CandidateList(): JSX.Element {
+
+    //TODO: use getCandidates function to get actual data
+    
+    const candidates: string[] = ["Michael Walker", "Stephen Jackson"];
```

Comment:
same here - if we create a little model for a Student with a Name and an Id and use that instead then we'll probably have to make far fewer changes later on!

Reply:
changed!